tests/integ/utils/sql_counter.py

```diff
@@ -376,17 +374,8 @@ def sql_count_checker(
     udf_count=None,
     udtf_count=None,
     union_count=None,
-    *args,
```

Comment:
The failure 

"sql_count_checker() got an unexpected keyword argument 'fallback_count'"

happens because the *args, **kwargs are removed here.

Not sure if that was intentional.  When I add them back, then the test_sql_counter.py tests pass.

Reply:
Thanks for this. I assumed these were `args` and `kwargs` for the `func` we are decorating so I moved them down to `wrapper`. I moved it back up now.